lib/teiserver/libs/teiserver_configs.ex

```diff
@@ -572,7 +572,7 @@ defmodule Teiserver.TeiserverConfigs do
       key: "profile.Rank method",
       section: "Profiles",
       type: "select",
-      default: "Leaderboard rating",
+      default: "Role",
```

Comment:
At one point the chevs were calculated by leaderboard rating. Now they are calculated using time. Search teiserver codebase for
```
calculate_rank(userid, "Role")
``` 
and you'll find the current algo. This algo can be changed via the admin panel and will effect the database.
http://localhost:4000/teiserver/admin/site
Go to profile and you can set the way chevs are calculated.